substratevm/mx.substratevm/suite.py

```diff
@@ -1062,7 +1059,6 @@
                 "com.oracle.svm.core.genscavenge",
                 "com.oracle.svm.core.containers",
                 "com.oracle.svm.jni",
-                "com.oracle.svm.jfr",
```

Comment:
Also here, should there be two additional entries for `core.jfr` and `hosted.jfr`?